### PR TITLE
Support JSON5

### PR DESCRIPTION
**Pull request by blond**

### CODE REVIEW

File: lib/file-contents-eval.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var path = require('path');
+
+var nodeEval = require('node-eval');
+var JSON5 = require('json5');
```
Comment (qfox) [inline lib/file-contents-eval.js:6]: What is faster, vm.runInThisContext or JSON5.parse?

**Review by birhoff — APPROVED** [review]

File: lib/file-contents-eval.js (hunk 1 above)
Comment (blond) [reply lib/file-contents-eval.js:6]: Run in 100000 files

Format | Real time | User Time | Sys Time
------- | --------- | ---------- | ----------
CommonJS | 4.237s | 3.709s | 1.359s
JSON | 4.240s | 3.724s | 1.367s
JSON 5 | 4.444s | 3.912s | 1.401s

**Review by tadatuta — APPROVED** [review]

File: lib/file-contents-eval.js (hunk 1 above)
Comment (qfox) [reply lib/file-contents-eval.js:6]: Ты воздух гонял, ты это знаешь?

Comment (blond) [reply lib/file-contents-eval.js:6]: просвети )

**Pull request merged**